app/js/components/orderEntry/OrderEntryPage.jsx

```diff
@@ -1,17 +1,47 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 import PatientDashboard from '../patientDashboard';
-
+import fetchPatientCareSetting from '../../actions/careSetting';
 import SearchAndAddOrder from './SearchAndAddOrder';
+import ActiveOrders from './ActiveOrders';
+
+export class OrderEntryPage extends React.Component {
+  componentDidMount() {
+    this.props.fetchPatientCareSetting();
+  }
 
-class OrderEntryPage extends React.Component {
   render() {
+    if (!(this.props.inpatientCareSetting || this.props.outpatientCareSetting)) {
+      return (
+        <div>
+          loading...
+        </div>
+      );
+    }
+
     return (
       <div>
+        <div />
         <PatientDashboard {...this.props} />
         <SearchAndAddOrder />
+        <ActiveOrders />
       </div>
     );
   }
 }
 
-export default OrderEntryPage;
+const mapStateToProps = ({ careSettingReducer }) => ({
+  inpatientCareSetting: careSettingReducer.patientCareSetting,
+  outpatientCareSetting: careSettingReducer.patientCareSetting,
+});
```

Comment:
is there a reason why you separated the patientCareSettings, if they're both receiving the same data from the reducer?

Reply:
Have modified my implementation to allow for fetching `inpatientCareSetting` and `outpatientCareSetting` explicitly. 
Thanks for pointing that out.